docs-new/docs/contribute/docs/spell-check.md

```diff
@@ -0,0 +1,76 @@
+# Spell checker
```

Comment:
```suggestion
# Spell Checker
```

Reply:
I thought we had agreed to use sentence case rather than title case for headers.  That is what both the Dynatrace and Google style guides recommend.  But I see that you have used Title Case elsewhere so I am implementing this until we can discuss again.

Reply:
ok then the nav section entry needs to be adjusted instead :)